diff --git a/crosshairs.kak b/crosshairs.kak
index 95f6491..a28c35d 100644
--- a/crosshairs.kak
+++ b/crosshairs.kak
@@ -1,5 +1,5 @@
-set-face global crosshairs_line default,rgb:383838+bd
-set-face global crosshairs_column default,rgb:383838+bd
+set-face global crosshairs_line default,rgb:383838
+set-face global crosshairs_column default,rgb:383838
 
 declare-option -hidden bool highlight_current_line false
 declare-option -hidden str highlight_current_line_hook_cmd "nop"
